data/server_locations.json

```diff
@@ -31172,13 +31173,13 @@
                 "area": "Germany",
                 "address": "Großer Feldberg 1, 61389 Schmitten, Germany",
                 "status": "unvisited",
-                "external_url": "http://elongated-coin.de/phpBB3/viewtopic.php?t=1767",
+                "external_url": "null",
```

Comment:
```suggestion
                "external_url": "http://elongated-coin.de/phpBB3/viewtopic.php?t=1767",
```

Reply:
Somehow I cannot commit your change with the link. Everytime I try to click on "Commit change", it's saying "This diff has recently been updated. Refresh and try again". Refreshing the page and trying againd did not help. Can you commit yourself?